helm/oncall/values.yaml

```diff
@@ -96,13 +98,27 @@ oncall:
     password: ~
     tls: ~
     fromEmail: ~
+  twilio:
+    # Twilio account SID/username to allow OnCall to send SMSes and make phone calls
+    accountSid: ~
+    # Twilio password to allow OnCall to send SMSes and make calls
+    authToken: ~
+    # Number from which you will receive calls and SMS
+    phoneNumber: ~
```

Comment:
Please specify that this _value must be in quotes_ otherwise it has int type and rendered as ` 1.234567e+10`

Reply:
Sorry, forgot about that thing! (We've faced the same issue :))